Snippets/WebsocketTest.swift

```diff
@@ -1,24 +1,28 @@
 import HTTPTypes
 import Hummingbird
 import HummingbirdWebSocket
+import Logging
 
+var logger = Logger(label: "Echo")
+logger.logLevel = .trace
 let router = Router(context: BasicWebSocketRequestContext.self)
 router.middlewares.add(FileMiddleware("Snippets/public"))
 router.get { _, _ in
     "Hello"
 }
 
-router.ws("/ws") { inbound, outbound, _ in
-    for try await packet in inbound {
+router.ws("/ws") { ws, _ in
+    for try await packet in ws.inbound {
```

Comment:
I like that this follows the pattern found in NIO. But I wonder if `ws.inbound` as a symbol is easy to reason about for newcomers to the ecosystem.